pages/public_cloud/containers_orchestration/managed_kubernetes/configuring-multi-attach-persistent-volumes-with-ovh-nas-ha/guide.fr-fr.md

```diff
@@ -40,10 +64,19 @@ $ kubectl get nodes -o jsonpath='{ $.items[*].status.addresses[?(@.type=="Intern
 51.77.204.175 51.77.205.79
 ```
 
+#### Your cluster is installed with Private Network with a gateway
+
+Because your nodes are configured to be routed by the private network gateway, you need to allow the gateway IP address to the ACLs.
```

Comment:
```suggestion
Because your nodes are configured to be routed by the private network gateway, you need to add the gateway IP address to the ACLs.
```